api/handler/service.go

```diff
@@ -1942,14 +1942,6 @@ func (s *ServiceAction) GetMultiServicePods(serviceIDs []string) (*K8sPodInfos,
 			podNames = append(podNames, v.PodName)
 			podsInfoList = append(podsInfoList, &podInfo)
 		}
-		containerMemInfo, _ := s.GetPodContainerMemory(podNames)
```

Comment:
Line 1938 should delete it? If deleted, will there be any impact on the upper console?

Reply:
done

Reply:
It will not affect the upper console.